src/main/java/nl/hsac/fitnesse/fixture/slim/DateFormatFixture.java

```diff
@@ -0,0 +1,73 @@
+package nl.hsac.fitnesse.fixture.slim;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.TimeZone;
+
+/**
+ * Fixture that helps converting dates to and from various formats. Intended to be used as a utility fixture i.e. as library
+ */
+
+public class DateFormatFixture extends SlimFixture {
+    private String dateFormat = "dd-MM-yyyy";
```

Comment:
I thought about using local defaults for timezone and format but decided to use defaults, beacouse using system settings yields unexpected results. For instance in one of my current projects, most testers create tests on their local machine, that has a dutch locale and a western europe timezone. The Jenkins master is a local server with a US locale and a western europe timezone and some jenkins slaves are EC2 instances with a US locale and a different timezone.

Setting a default ensures similar behaviour wherever the test runs. Your thoughts on that?